webapp/app/translations/de/LC_MESSAGES/messages.po

```diff
@@ -2472,23 +2472,23 @@ msgstr ""
 #: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:242
 msgid "form.lotse.steuerminderungen.details-list-item-1"
 msgstr ""
-"Es muss eine Rechnung vorliegen, die zum Beispiel per Überweisung oder "
-"EC-Kartenzahlung beglichen worden ist. Barzahlungen können nicht geltend "
-"gemacht werden."
+"Es <b>muss eine Rechnung</b> vorliegen, die zum Beispiel per Überweisung "
+"oder EC-Kartenzahlung beglichen worden ist. <b>Barzahlungen können "
+"nicht</b> geltend gemacht werden."
 
 #: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:243
 msgid "form.lotse.steuerminderungen.details-list-item-2"
 msgstr ""
-"Abgesetzt werden können Kosten, die Sie im Besteuerungsjahr bezahlt "
-"haben. Relevant ist der Zeitpunkt der Bezahlung, nicht der "
+"Abgesetzt werden können Kosten, die Sie <b>im Besteuerungsjahr "
+"bezahlt</b> haben. Relevant ist der Zeitpunkt der Bezahlung, nicht der "
 "Rechnungsstellung."
 
 #: app/forms/steps/lotse_multistep_flow_steps/steuerminderungen_steps.py:244
 msgid "form.lotse.steuerminderungen.details-list-item-3"
 msgstr ""
-"Wenn Sie sich als Arbeitgeber betätigen und Haushaltshilfen als "
-"Arbeitnehmer einstellen, können Sie dies im Steuerlotsen nicht steuerlich"
-" absetzen."
+"Wenn Sie sich als <b>Arbeitgeber betätigen und Haushaltshilfen</b> als "
+"Arbeitnehmer einstellen, können Sie dies im Steuerlotsen <b>nicht "
+"steuerlich absetzen</b>."
```

Comment:
Please don't use `<b>`, see here for an explanation: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

You can use `<strong>` instead.